_posts/2022-07-18-ols5-speedblog-open-science-mooc.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "OLS-5 speedblog: Open Science MOOC, TU Delft"
+authors:
+- AleCandian
+- lwbwalma
+image: /images/2022-07-18-osmooc-logo.png
+---
+
+## Project background
+
+The Delft University of Technology (TU Delft) in the Netherlands runs once a year a Massive Online Open Course (MOOC) on Open Science on the edX platform => [Open Science: Share your research with the world](https://learning.edx.org/course/course-v1:DelftX+OS101x+2T2022/home).  Our primary target is PhD candidates but the MOOC is meant also for MSc students, researchers, support staff, policy makers that want to learn the basic principles of and motivations behind Open Science.  
+The OSMOOC has been running for multiple years with a decreasing number of enrollments and varying levels of engagement. We have high dropout rates and we do not know if the participants practice Open Science later on. Our project goal was to improve the engagement by creating a community around the MOOC participants.
+
+![Logo of the Open Science MOOC](/images/2022-07-18-osmooc-logo.png)![titles of the six modules contained in the course: Introduction to Open Science, Open Data, Open Access Publishing, Visibility as a Researcher, FAIR Software, and Citizen Science](/images/2022-07-18-osmooc-modules.png)
+
+_Figure1: Logo of the Open Science MOOC and titles of the six modules contained in the course. Image by TU Delft and licensed under [CC-BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) International License._
+
+## Expectations from this program
+
+Since we had both just started in our jobs as open science teachers, the first thing we hoped to learn was more about the different themes within Open Science. We also hoped that through our participation in the OLS program, we would learn more ways to engage our MOOC’s participants and build a solid community, as well as workflows and processes to make our own work more open.
```

Comment:
```suggestion
Since we had both just started in our jobs as open science teachers, the first thing we hoped to learn was more about the different themes within Open Science. We also hoped that through our participation in the OLS program, we would learn more ways to engage our MOOC's participants and build a solid community, as well as workflows and processes to make our own work more open.
```